modules/infinite-scroll/infinity.php

```diff
@@ -1538,9 +1538,19 @@ function footer() {
 	 *
 	 * @uses __, wp_get_theme, apply_filters, home_url, esc_attr, get_bloginfo, bloginfo
 	 * @return string
+	 *
 	 */
 	private function default_footer() {
-		$credits = sprintf(
+		if ( get_privacy_policy_url() !== '' ) {
+			$credits = sprintf(
+				'<a href="%1$s" rel="noopener noreferrer" target="_blank">%2$s</a><span role="separator" aria-hidden="true"> / </span>',
+				get_privacy_policy_url(),
+				__('Privacy Policy', 'jetpack' )
```

Comment:
Missing space between '(' & 'Privacy Policy'

Reply:
@brbrr Oops! I can push another commit right now. I assume I should amend the last one so it's not just a typo fix?

Reply:
No worries about number of commits. We are squashing before merge. Thanks!